app/src/main/kotlin/com/google/samples/apps/nowinandroid/NiaApplication.kt

```diff
@@ -28,9 +26,7 @@ import javax.inject.Inject
  * [Application] class for NiA
  */
 @HiltAndroidApp
-class NiaApplication : Application(), ImageLoaderFactory {
```

Comment:
It is used when we do not specify an ImageLoader in Coil AsyncImage Composable

Reply:
https://github.com/coil-kt/coil/blob/main/coil-compose%2Fsrc%2FcommonMain%2Fkotlin%2Fcoil3%2Fcompose%2FSingletonAsyncImage.kt#L68-L68

Reply:
https://github.com/coil-kt/coil/blob/main/coil%2Fsrc%2FcommonMain%2Fkotlin%2Fcoil3%2FSingletonImageLoader.common.kt#L111-L113